ui/src/literature/components/CiteAllAction.jsx

```diff
@@ -24,21 +22,18 @@ class CiteAllAction extends Component {
   async onCiteClick({ key }) {
     const { query } = this.props;
     const citeQuery = {
-      // set default `sort` in case there is no `sort` in the query
-      sort: searchScopes.getIn(['literature', 'query', 'sort']),
-      ...omit(query, ['size', 'page']),
+      ...query,
```

Comment:
this is wrong, it should omit the page

Reply:
I don't get why it would be wrong?, `page=1` is already a default on the backend.

Reply:
ah, missed the fact that you added ```page=1``` after it. Thought if you were on page 2 and pressed on ```citeAll`` it won't give results, my bad